hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1408,6 +1411,8 @@ protected void commitInternal(String instantTime, Map<String, HoodieData<HoodieR
       bulkInsertAndCommit(writeClient, instantTime, preppedRecordInputs, bulkInsertPartitioner);
     } else {
       engineContext.setJobStatus(this.getClass().getSimpleName(), String.format("Upserting at %s into metadata table %s", instantTime, metadataWriteConfig.getTableName()));
+      // to do: fix the last argument is required so that we can support streaming writes to metadata table.
+      // Option.of(partitionFileIdPairs)
```

Comment:
if this is handled in a later PR, my suggestion is to just remove the TODO from here